### PR TITLE
Dbqueries

### PR DESCRIPTION
**Pull request by nicolemon**

### CODE REVIEW

**Comment by austinpray** [comment]
Will review soon

File: main.go [hunk 1]
```diff
@@ -70,37 +78,193 @@ func (fw *flushWriter) Write(p []byte) (n int, err error) {
...
+	err = json.Unmarshal(dbjson, &dbvars)
+	check(err)
+
+	dbcreds := dbvars
+	dbinfo := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable", dbcreds.DB_HOST, dbcreds.DB_PORT, dbcreds.DB_USER, dbcreds.DB_PASSWORD, dbcreds.DB_NAME)
+	db, err := sql.Open("postgres", dbinfo)
+	check(err)
+	defer db.Close()
+
+	stmt, err := db.Prepare("CREATE TABLE chunks_a AS SELECT * FROM channel_chunks WHERE met_end > $1")
```
Comment (austinpray) [inline main.go:153]: wait wut. Why is this creating tables?

Comment (nicolemon) [inline main.go:153]: Temporary tables to get all the audio clips that fall in the requested time. First one gets all chunks that end after the start time, thus eliminating all chunks to the left of the request, then of the remaining chunks those that end to the right of the request. Picture related.
img_20151219_163142647
We could probably do this without creating tables, but my relational algebra game is weak so halp.

Comment (austinpray) [inline main.go:153]: Yeah we def can't be creating temp tables. That's race condition city. I'll think about it.

**Pull request closed without merging**